docs/zh-cn/1.2.0/user_doc/standalone-deployment.md

```diff
@@ -0,0 +1,451 @@
+    ```
+
+ - 修改一键部署脚本 `install.sh`中的各参数，特别注意以下参数的配置
+
+    ```shell
+    # 这里填 mysql or postgresql
+    dbtype="mysql"
+    
+    # 数据库连接地址
+    dbhost="localhost:3306"
+    
+    # 数据库名
+    dbname="dolphinscheduler"
+    
+    # 数据库用户名，此处需要修改为上面设置的{user}具体值
+    username="xxx"    
+    
+    # 数据库密码, 如果有特殊字符，请使用\转义，需要修改为上面设置的{passowrd}具体值
+    passowrd="xxx"
+    
+    #将DS安装到哪个目录，如: /opt/soft/dolphinscheduler，不同于现在的目录
+    installPath="/opt/soft/dolphinscheduler"
+    
+    #使用哪个用户部署，使用1.3小节创建的用户
+    deployUser="dolphinscheduler"
+    
+    #zookeeper地址，单机本机是localhost:2181，记得把2181端口带上
+    zkQuorum="localhost:2181"
+    
+    #在哪些机器上部署DS服务，本机选localhost
+    ips="localhost"
+    
+    #master服务部署在哪台机器上
+    masters="localhost"
+    
+    #worker服务部署在哪台机器上
+    workers="localhost"
+    
+    #报警服务部署在哪台机器上
+    alertServer="localhost"
+    
+    #后端api服务部署在在哪台机器上
+    apiServers="localhost"
+    
+    
+    # 邮件配置，以qq邮箱为例
+    # 邮件协议
+    mailProtocol="SMTP"
+    
+    # 邮件服务地址
+    mailServerHost="smtp.exmail.qq.com"
+    
+    # 邮件服务端口
+    mailServerPort="25"
+    
+    # mailSender和mailUser配置成一样即可
+    # 发送者
+    mailSender="xxx@qq.com"
+    
+    # 发送用户
+    mailUser="xxx@qq.com"
+    
+    # 邮箱密码
+    mailPassword="xxx"
+    
+    # TLS协议的邮箱设置为true，否则设置为false
+    starttlsEnable="true"
+    
+    # 邮件服务地址值，参考上面 mailServerHost
+    sslTrust="smtp.exmail.qq.com"
+    
+    # 开启SSL协议的邮箱配置为true，否则为false。注意: starttlsEnable和sslEnable不能同时为true
+    sslEnable="false"
+    
+    # excel下载路径
+    xlsFilePath="/tmp/xls"
+    
+    # 业务用到的比如sql等资源文件上传到哪里，可以设置：HDFS,S3,NONE，单机如果想使用本地文件系统，请配置为HDFS，因为HDFS支持本地文件系统；如果不需要资源上传功能请选择NONE。强调一点：使用本地文件系统不需要部署hadoop 
+    resUploadStartupType="HDFS"
+    
+    # 这里以保存到本地文件系统为例
+    #注：但是如果你想上传到HDFS的话，NameNode启用了HA，则需要将core-site.xml和hdfs-site.xml放到conf目录下，本例即是放到/opt/dolphinscheduler/conf下面，并配置namenode cluster名称；如果NameNode不是HA,则修改为具体的ip或者主机名即可 
+    defaultFS="file:///data/dolphinscheduler"    #hdfs://{具体的ip/主机名}:8020
+    
+    
+    # 如果ResourceManager是HA，则配置为ResourceManager节点的主备ip或者hostname,比如"192.168.xx.xx,192.168.xx.xx"，否则如果是单ResourceManager或者根本没用到yarn,请配置yarnHaIps=""即可，我这里没用到yarn，配置为""
+    yarnHaIps=""
+    
+    # 如果是单ResourceManager，则配置为ResourceManager节点ip或主机名，否则保持默认值即可。我这里没用到yarn，保持默认
+    singleYarnIp="ark1"
+    
+    # 由于hdfs支持本地文件系统，需要确保本地文件夹存在且有读写权限
+    hdfsPath="/data/dolphinscheduler"
+    ```
+    
+    *注：如果打算用到`资源中心`功能，请执行以下命令：*
+    
+    ```shell
+    sudo mkdir /data/dolphinscheduler
+    sudo chown -R dolphinscheduler:dolphinscheduler /data/dolphinscheduler
+    ```
+
+### 1.7 : 安装python的zookeeper工具kazoo
+
+- 安装python的 zookeeper 工具 ，`此步骤仅在一键部署时候用到`
+
+```shell
+#安装pip
+sudo yum -y install python-pip;  #ubuntu请使用 sudo apt-get install python-pip
+sudo pip install kazoo;
+```
+
+  *注意：如果yum没找到python-pip，也可以通过下面方式安装*
+
+```shell
+sudo curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+sudo python get-pip.py  # 如果是python3，使用sudo python3 get-pip.py 
+#然后
+sudo pip install kazoo;
+```
+
```

Comment:
this step is optional

Reply:
which step is optional?

Reply:
1.2.0 need this step, from 1.2.1, kazoo is removed